### PR TITLE
[DONE] Translating complex-and-rational-numbers.md

### PR DESCRIPTION
**Pull request by ghost**
# 🎉 **FINISHED** 🎉 
#### Translation progress:
- ✅  *__Complex and Rational Numbers__* section
- ✅  *__Complex Numbers__* section
- ✅  *__Rational Numbers__* section

### CODE REVIEW

**Comment by hshindo** [comment]
Translation of julia-v0.5 doc is here:
https://github.com/hshindo/julia-doc-ja/tree/master/doc/manual
including `complex-and-rational-numbers`.  You need to convert `rst` to `md`, though.

**Comment by ghost** [comment]
Thank you for replying. Now, I think all that I should do is to translate modification between v0.5 and v0.6.

**Comment by hshindo** [comment]
Thank you.
Since the translations are not perfect, it would also be welcome to correct the translation errors.

File: doc/src/manual/complex-and-rational-numbers.md [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
+
+[](
 The global constant [`im`](@ref) is bound to the complex number *i*, representing the principal
 square root of -1. It was deemed harmful to co-opt the name `i` for a global constant, since it
 is such a popular index variable name. Since Julia allows numeric literals to be [juxtaposed with identifiers as coefficients](@ref man-numeric-literal-coefficients),
 this binding suffices to provide convenient syntax for complex numbers, similar to the traditional
 mathematical notation:
+)
+
+グローバル定数[`im`](@ref)は-1の平方根を表す複素数 *i* に紐付きます。 ``i`` は一般的なインデックス変数であるため、
```
Comment (ghost) [inline doc/src/manual/complex-and-rational-numbers.md]: "グローバル定数[`im`](@ref)は-1の平方根を表す複素数 *i* に紐付きます。なぜ[`i`]としないのかというと、 ``i`` は一般的なインデックス変数であるため..."の方がより良い気がします。

**Comment by ghost** [comment]
I have done translating complex-and-rational-numbers.md.
I believe there is no typo, but there **could** be. if you find any typo, please correct it.

**Pull request merged**